server/lib/client-html.ts

```diff
@@ -549,6 +549,7 @@ class ClientHtml {
 
 function sendHTML (html: string, res: express.Response) {
   res.set('Content-Type', 'text/html; charset=UTF-8')
+  res.set('Vary', 'Accept-Language')
 
```

Comment:
I think you should also use `Cookie` for `Vary` since we can also serve a different HTML page depending on the user cookie.

Reply:
In what cases are the HTML served differently based on the cookie? According to this docs it's not recommended to use Vary Cookie; https://www.fastly.com/blog/best-practices-using-vary-header

Reply:
> In what cases are the HTML served differently based on the cookie?

When the user explicitly set the language in the interface: https://github.com/Chocobozzz/PeerTube/blob/develop/server/lib/client-html.ts#L335 (to remember to serve this explicit lang in future index HTML calls)

> According to this docs it's not recommended

Peertube only uses one cookie that stores client language. But since plugins like matomo can also store cookies, I agree we may not add it.

Reply:
Hmm. That was a hard case. For most of the time that shouldn't be a problem for the user, unless there's a CDN in the picture. But I think that most CDN's has the possibility to bypass cache based on a specific cookie. Maybe we can add something about this in the documentation?

Reply:
Yep